jdbc/core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java

```diff
@@ -57,7 +62,25 @@ public ShardingSphereDriverURL(final String url) {
     @SneakyThrows(IOException.class)
     public byte[] toConfigurationBytes() {
         try (InputStream stream = inClasspath ? ShardingSphereDriverURL.class.getResourceAsStream("/" + file) : Files.newInputStream(new File(file).toPath())) {
-            byte[] result = new byte[null == stream ? 0 : stream.available()];
+            LineProcessor<byte[]> lineProcessor = new LineProcessor<byte[]>() {
+                
+                private final StringBuilder builder = new StringBuilder();
+                
+                @Override
+                public boolean processLine(final String line) {
+                    if (line.startsWith("#") || 0 == line.length()) {
+                        return true;
+                    }
```

Comment:
Looks empty line is removed by `0 == line.length()`. It's better to keep empty line, since it's the content of configuration file.